### PR TITLE
Update private key jwt reuse app config

### PR DESCRIPTION
**Pull request by mpmadhavig**
1. New config adding steps added.
2. Org level api removed.
3. 7.0.0 docs were separated from the template.

Please note that asgardeo didn't have API support.

### Related Issues:
https://github.com/wso2/product-is/issues/20546

### CODE REVIEW

File: en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md [hunk 1]
```diff
@@ -213,14 +227,32 @@ Be sure to replace the following values in the request:
...
 ## Reuse tokens (optional)
 
 The `jti` (JWT ID) claim is a unique identifier included in the JWT token, which controls the reuse of the access token. By default, token reuse is disabled in {{ product_name }}. If you enable token reuse, the `jti` can be reused within its expiration period.
 
 To enable token reuse in {{ product_name }}.
 
-1. On the Asgardeo Console, go to **Login & Registration**.
-2. Under **Login Security**, click **Private Key JWT Client Authentication (OIDC)**.
-3. Switch on the toggle to enable token reuse.
+1. On the {{ product_name }} Console, go to **Organizational Settings** > **Account Security > Private Key JWT Client Authentication for OIDC**.
```
Comment (himeshsiriwardana) [inline en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md:238]: These instructions are for IS 7? If so, the original ones are accurate aren't they? The updated ones are deprecated?

Comment (mpmadhavig) [reply en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md:238]: Here only the Asgardeo console need to be replaced with {{ product_name }}. Will update the PR with that.

Comment (mpmadhavig) [reply en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md]: Fixed with 125b6353d3ca3c1719293beaf109e4b71772ff62

File: en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md [hunk 2]
```diff
@@ -29,10 +29,24 @@ Register the client application in {{ product_name }} as follows:
...
+
+    ![oidc protocols]({{base_path}}/assets/img/guides/applications/oidc/oidc_protocols.png)
+
+{% else %}
+
 2. Go to the **Protocol** tab of the new application and configure the required grant type.
 
     ![oidc protocols]({{base_path}}/assets/img/guides/applications/oidc/oidc_protocols.png){: width="700" style="border: 0.3px solid lightgrey;"}
 
+3. In the **Protocol** tab, go to the **Client Authentication** section and under **Client authentication method**, select **Private Key JWT** from the dropdown.
```
Comment (himeshsiriwardana) [inline en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md:44]: let's break this down to 2 steps.

**Review by himeshsiriwardana — APPROVED** [review]

**Comment by mpmadhavig** [comment]
Related PR: https://github.com/wso2/docs-is/pull/4704

**Pull request merged**